app/assets/stylesheets/custom.css.scss

```diff
@@ -33,3 +33,8 @@ ul.site-links {
 .nav .active a {
   border-bottom: 4px solid #47a447;
 }
+
+.form-group.user_remember_me {
```

Comment:
There is no need in a dedicated style, we can achieve the same behaviour using the Bootstrap's existing classes. There is an example https://github.com/ck3g/homebugh/blob/5ecb501407dcd2106467af16e20f34c93504beb6/app/views/categories/_form.html.haml#L4-L7

Reply:
Ohhh got it, sorry for the trouble. I'm super new to this so I did this. Thank you so much! Will make the changes and also remove the files you mentioned from the commit.

Reply:
Sure, no problem at all.

Reply:
@ck3g I have made some changes, not sure if they're perfect, but please check it out and let me know. Thank you!